src/plugins/curlget/README.md

```diff
@@ -5,7 +5,7 @@
 - infos/provides =
 - infos/recommends =
 - infos/placements = getresolver setresolver commit rollback
-- infos/status = configurable readonly preview unfinished nodoc
+- infos/status = configurable readonly preview unfinished
```

Comment:
I don't remember that the documentation was improved there? I don't think we can rely on the heuristics of the update tool, please verify manually (or keep as-is).

Reply:
Well, this plugin actually seems to have extensive documentation at a first glance.